Apps/Examples/Examples.xcodeproj/project.pbxproj

```diff
@@ -204,6 +205,7 @@
 		58A3C0C825C4B93600CAE5F0 /* AnimateGeoJSONLineExample.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = AnimateGeoJSONLineExample.swift; sourceTree = "<group>"; };
 		666E0D4925664EE7000B8AF5 /* LayerPositionExample.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = LayerPositionExample.swift; sourceTree = "<group>"; };
 		732A54FF28DA470E00CDBFB7 /* PitchAndDistanceExample.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = PitchAndDistanceExample.swift; sourceTree = "<group>"; };
+		734CD68728F9C32C001B1F8E /* PointAnnotationClusteringExample.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; name = PointAnnotationClusteringExample.swift; path = "../../../../../../../mapbox-maps-ios/Apps/Examples/Examples/All Examples/Annotations/PointAnnotationClusteringExample.swift"; sourceTree = "<group>"; };
```

Comment:
the path seems to be wrong. It should be just `Annotations/PointAnnotationClusteringExample.swift`

Reply:
Good catch -- updated and should be resolved now